---
cob_object_detection_msgs/DetectionArray object_list
